docs/sphinx/using/install/install.rst

```diff
@@ -893,20 +893,27 @@ you can set the size of the secret string to up to 28-32 when running on the `nv
 
 Let's compare that to using only your CPU:
 
-.. tab:: C++
+.. tab:: Python
 
   .. code-block:: console
 
-      nvq++ examples/cpp/algorithms/bernstein_vazirani.cpp -DSIZE=25 --target qpp-cpu && ./a.out
+      python examples/python/bernstein_vazirani.py --size 25 --target qpp-cpu
 
-.. tab:: Python
+.. tab:: C++
 
   .. code-block:: console
 
-      python examples/python/bernstein_vazirani.py --size 25 --target qpp-cpu
+      nvq++ examples/cpp/algorithms/bernstein_vazirani.cpp -DSIZE=25 --target qpp-cpu && ./a.out
 
 When you execute this command, the program simply seems to hang; that is because it takes
 a long time for the CPU-only backend to simulate 28+ qubits! Cancel the execution with `Ctrl+C`.
 
 You are now all set to start developing quantum applications using CUDA Quantum!
-Please proceed to :doc:`Using CUDA Quantum <using/cudaq>` to learn the basics.
+Please proceed to :doc:`Basics <../basics>` for an introduction
+to the fundamental features of CUDA Quantum.
+
+
+Data Center Installation
```

Comment:
Ah, that explains why you changes the section hierarchy in the other document. I think the Installation from source needs to be a separate page, since the vast majority of the people should never look at it, but a good number of people sooner or later should read the install page. I don't think this is a good page to absorb this into one gigantic page.

Reply:
I've updated the index structure as follows:

```
> Quick Start
.
.
.
> Installation
      >> Local Installation
            >>>> Introduction
            >>>> ...
      >> Data Center Installation
            >>>> Installation from Source
            >>>> ...
```